src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json

```diff
@@ -23,17 +23,19 @@
             "in": "path",
             "required": true,
             "schema": { "type": "string" }
+          },
+          {
+            "name": "checkVTCAssigned",
+            "in": "path",
+            "required": true,
+            "schema": { "type": "boolean" }

```

Comment:
Is there a reason why you made this boolean a path variable (on the endpoint path) instead of a regular query/request parameter? This doesn't really follow RESTApi semantics. Path variables are usually used to separate resources (objects).

Reply:
I will try putting it in the body of the request